apps/ex_wire/lib/ex_wire/adapter/tcp/server.ex

```diff
@@ -1,282 +0,0 @@
-defmodule ExWire.Adapter.TCP.Server do
```

Comment:
Most of the logic in each of the functions of this module now lives in `P2P`. I essentially extracted the body of all the functions from this module into `P2P`.